style/javascript.md

```diff
@@ -552,3 +553,53 @@ Yes:
+delay | `setTimeout(fn, 2000);` | `_.delay(fn, 2000);`
+each | `array.forEach(fn)` | `_.each(array, fn)`
+each | `for (const val of array) {}` | `_.each(array, fn)`
+each | `for (const [key, val] of Object.entries(obj)) {}` | `_.each(obj, fn)`
+extend | `{...options, prop: 1}` | `_.extend({}, options, {prop: 1})`
+extend | `{...defaultOptions, ...options}` | `_.extend(defaultOptions, options || {})`
+extend | `Object.assign(json, this.model.toJSON())` | `_.extend(json, this.model.toJSON())`
+filter | `array.filter(checkFn)` | `_.filter(array, checkFn)`
+has | `array.includes(value)` | `_.has(array, value)`
+has | `obj.hasOwnProperty(value)` <sup>[3](#u3)</sup> | `_.has(obj, value)`
+isArray | `Array.isArray(someObj)` | `_.isArray(someObj)`
+isFunction | `typeof fn === "function"` | `_.isFunction(fn)`
+isString | `typeof obj === "string"` | `_.isString(obj)`
+keys | `Object.keys(obj)` | `_.keys(obj)`
+last | `someArray[someArray.length - 1]` | `_.last(someArray)`
+last | `someArray.pop()` <sup>[4](#u4)</sup> | `_.last(someArray)`
+map | `array.map(mapFn)` | `_.map(array, mapFn)`
+max | `Math.max(...array)` | `_.max(array)`
+omit | `array.filter(prop => !props.includes(prop))` | `_.omit(array, props)`
+omit | <pre>Object.keys(obj).reduce((result, prop) => {<br>&nbsp;&nbsp;&nbsp;&nbsp;if (!props.includes(prop)) {<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;result[prop] = attrs[prop];<br>&nbsp;&nbsp;&nbsp;&nbsp;}<br>}, {})</pre> | `_.omit(obj, props)`
+once | `$(...).one("click", ...)` | `$(...).on("click", _.once(...))`
+once | <pre>{<br>&nbsp;&nbsp;&nbsp;&nbsp;method: () => {<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;if (this._initDone) { return; }<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;this._initDone = true;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;...<br>&nbsp;&nbsp;&nbsp;&nbsp;}<br>}</pre>| `{ method: _.once(() => { ... }) }`</pre>
+once | <pre>var getResult = () => {<br>&nbsp;&nbsp;&nbsp;&nbsp;let val = $.when(...).then(...);<br>&nbsp;&nbsp;&nbsp;&nbsp;getResult = () => val;<br>&nbsp;&nbsp;&nbsp;&nbsp;return val;<br>};</pre> | <pre>var getResult = _.once(() => {<br>&nbsp;&nbsp;&nbsp;&nbsp;return $.when(...).then(...);<br>});</pre>
+object | <pre>Object.entries(obj).reduce((result, [key, val]) => {<br>&nbsp;&nbsp;&nbsp;&nbsp;result[key] = value;<br>&nbsp;&nbsp;&nbsp;&nbsp;return result;<br>})</pre> | <pre>_.object(_.map(obj, (val, key) => {<br>&nbsp;&nbsp;&nbsp;&nbsp;return [key, value];<br>})</pre>
+sortBy | `result = result.sort((a, b) => a.prop - b.prop)` | `_.sortBy(result, "prop")`
+sortedIndex | Our custom lodash build. <sup>[2](#u2)</sup> |
+throttle | Our custom lodash build. <sup>[2](#u2)</sup> |
+values | `Object.values(obj)` | `_.values(obj)`
+
+1. Or use a loop if binding multiple methods. <b id="u1"></b>
+2. No alternative at the moment! If you need it then you should add it to the compiled version of lodash and then update this guide to mention that it now exists! <b id="u2"></b>
```

Comment:
I don't understand this footnote.  The text says this is in our custom lodash build, so what does 'no atlernative at the moment' refer to?  Are you saying we could add it to the lodash build but haven't yet?  (In these cases, I thought we had.)